test/common/testlib.py

```diff
@@ -556,6 +556,13 @@ def copy_js_log(self, title, label=None):
     def kill(self):
         self.cdp.kill()
 
+    def grant_permissions(self, origin, *args):
+        """Grant permissions to the browser"""
+        # https://chromedevtools.github.io/devtools-protocol/tot/Browser/#method-grantPermissions
+        self.cdp.invoke("Browser.grantPermissions",
+                        origin=origin,
+                        permissions=[x for x in args])
+
```

Comment:
It's the smallest thing, but would you mind moving this upwards a little bit, to similar methods? Right after `ignore_ssl_certificate_errors()` sounds right, as it's also permission related.

Also, I don't understand the `[x for x in args]` -- `args` is already a list, what does this convert?

Reply:
> `args` is already a list

args is tuple. and by "rather safe than sorry" I did the conversion, but now I tested it and it seems it works just fine with tuples, so I will remove it.

Reply:
Right, tuple, I forgot about that. You can write `list(args)` to be on the safe side.